packages/playwright/src/executors/playwright/playwright.ts

```diff
@@ -0,0 +1,183 @@
+import { exec, spawn } from 'child_process';
+import { ExecutorContext, getPackageManagerCommand } from '@nx/devkit';
+
+export async function playwrightExecutor(
+  options: PlaywrightExecutorSchema,
+  context: ExecutorContext
+) {
+  const projectRoot =
+    context.projectGraph?.nodes?.[context?.projectName]?.data?.root;
+
+  if (!projectRoot) {
+    throw new Error(
+      `Unable to find the Project Root for ${context.projectName}. Is it set in the project.json?`
+    );
+  }
+  const args = createArgs(options);
+  const p = runPlaywright(args, projectRoot);
+
+  return new Promise<{ success: boolean }>((resolve) => {
+    // TODO(caleb): are there other signals I need to care about?
+    p.on('close', (code) => {
+      resolve({ success: code === 0 });
+    });
+  });
+}
+
+function createArgs(opts: PlaywrightExecutorSchema): string[] {
+  //NOTE: boolean options send at --arg=false might be invalid options.
+  const args: string[] = [];
```

Comment:
i am just wondering whether this could be simplified, instead of listing all the options, could we loop through it?